### ISSUE TITLE
Invalid time value error

### ISSUE THREAD

**Issue by mtahirnawaz** [Hacker0x01/react-datepicker#1812]
**Error log**
Invalid time value
**Pass value fields**
selected={'03/07/2019'}
Using Version
`react-datepicker": "^2.7.0`

**Comment by jylopez** [comment]
I get the same error with `selected='2019-07-24 06:00:00'`

**Comment by jylopez** [comment]
@mtahirnawaz I figured it out. You have to put `selected={Date.parse('03/07/2019')}`

**Comment by yashSrijan** [comment]
life saver @jylopez

**Comment by darrenklein** [comment]
I encountered this problem as well, @jylopez fix worked for me, too. Just to illustrate the situation -

In one context, I was setting the value of `selected` simply as the result of `new Date()`, which would be something like:

```
Thu Mar 12 2020 14:47:32 GMT-0400 (Eastern Daylight Time)
```

In those cases, I **did not** need to `.parse()` the data.

However, sometimes I was populating the datepicker with data from the DB, which was received by the client as a UTC string like `2020-02-21T16:00:00.000Z`, which I was then passing to `new Date()`, which would return:

```
Fri Feb 21 2020 11:00:00 GMT-0500 (Eastern Standard Time)
```

In those cases, I **did** need to `.parse()` the data - I assume there's some discrepancy between `Eastern Daylight Time` and `Eastern Standard Time`. 

Anyway, I just made a point to parse whatever data the picker is receiving, and all is well.
